src/dataset-builder/DomainCriteriaSelector.test.ts

```diff
@@ -67,6 +79,6 @@ describe('DomainCriteriaSelector', () => {
     const user = userEvent.setup();
     await user.click(screen.getByLabelText('cancel'));
     // Assert
-    expect(onStateChange).toHaveBeenCalledWith(cohortEditorState.new(cohort));
+    expect(onStateChange).toHaveBeenCalledWith(state.cancelState);
```

Comment:
and just for clarity- in this case the cancel state is which AnyDatasetBuilderState? the cohort editor?

Reply:
The `cancelState` can be anything that conforms to `AnyDatasetBuilderState`; the test is asserting that the state change that occurs is causing a change to the provided cancel state, vs before when the cancel state was hardcoded and couldn't be changed by the caller.